modules/core-services/endpoints/project.go

```diff
@@ -81,7 +81,7 @@ func (e *Endpoints) UpdateProject(ctx context.Context, r *http.Request, vars map
 		return apierrors.ErrUpdateProject.NotLogin().ToResp(), nil
 	}
 	orgIDStr := r.Header.Get(httputil.OrgHeader)
-	orgID, err := strutil.Atoi64(orgIDStr)
+	_, err = strutil.Atoi64(orgIDStr)
```

Comment:
We should compare the equality of orgId in header with orgId in projectInfo. If they are not equal, a appropriate error should be returned (it is also permission denied because user beyond the boundary of org).

Reply:
make sense